scripts/named_entity_recognition/code/__init__.py

```diff
@@ -0,0 +1,18 @@
+# coding: utf-8
```

Comment:
BTW, a module named like `code` will [hide the stdlib module code](https://stackoverflow.com/questions/20157925/pycharm3-0-1win7-when-i-debug-my-webpy-project-it-output-importerror-cannot-im), disabling the debugger of PyCharm.